src/components/Dropdown/__stories__/dropdown.stories.mdx

```diff
@@ -695,6 +695,16 @@ component) or container with overflow hidden content.
 Use <b>insideOverflowWithTransformContainer</b> prop attribute for displaying drop down inside scrollable container which use CSS transform function (like our dialog component) or container
 with overflow hidden content which use CSS transform function.
 
+<Tip title="Devs tip">
+  For more details about dropdown's APIs for displaying correctly inside Dialogs, Modals and other popovers
+  <Link
+    href="?path=/docs/technical-patterns-dropdowns-inside-pop-overs--modal-with-damaged-dropdown"
```

Comment:
Link is not working correctly. I think `href` should start with `/?`, but check it, please